tree/dataframe/test/dataframe_snapshot.cxx

```diff
@@ -750,12 +750,10 @@ void ReadWriteCarray(const char *outFileNameBase)
       EXPECT_TRUE(r.Next());
       EXPECT_EQ(rv.GetSize(), 100000u);
       EXPECT_EQ(rvb.GetSize(), 100000u);
-      for (auto e : rv)
-         EXPECT_EQ(e, 84);
-      for (auto e : rvb)
-         EXPECT_TRUE(e);
-      for (auto e : rvl)
-         EXPECT_EQ(e, 42);
+      EXPECT_EQ(rvl.GetSize(), 100000u);
+      EXPECT_TRUE(std::all_of(rv.begin(), rv.end(), [](auto elm){ return elm == 84; })) << rv[0] << " " << rv[1] << " " << rv[2];
```

Comment:
Sorry I don't remember the details of this test, why are you printing rv[0], rv[1] and rv[2] specifically in case of failure?

Reply:
Previously, it was printing some 10k failure messages. Now it's only one message, and you see the first three examples. I admit that three is arbitrary, but it was a better number than 10k.